### PR TITLE
Feature#47design forms devise

### LINKED ISSUE #47: design devise forms
# REQUIREMENTS
- add design to forgot password view.
- add design to change password view.
- add design to confirm your email view.

# Acceptance Criteria
List all required acceptance criteria
- [ ] works correctly

### PR DESCRIPTION
**Pull request by DerekAyala**
# Description
What did you implement/Why did you implement this:

- design to forgot password
- design to change password
- design to resend confirmation

  ## Screenshots 
forgot password
Screenshot from 2022-02-22 16-49-26
change password
Screenshot from 2022-02-22 16-49-59
resend confirmation 
Screenshot from 2022-02-22 16-51-34

### CODE REVIEW

**Review by IrenehMDev — APPROVED** [review]
LGTM

File: app/assets/stylesheets/components/_forms.scss [hunk 1]
```diff
@@ -67,27 +63,35 @@
...
+}
 
-  &__errors {
-    display: grid;
-    gap: 0.5rem;
-    list-style: none;
-    margin-top: 1rem;
+.newForm__title {
+  text-align: center;
+}
```
Comment (YilKanda) [inline app/assets/stylesheets/components/_forms.scss]: Regarding these classes called **newForm**, what do you mean by that name? Remember that the more defined the name of your class, the better it is within your development.

File: app/assets/stylesheets/components/_forms.scss [hunk 2]
```diff
@@ -67,27 +63,35 @@
...
 
-    li {
-      font-size: 1.5rem;
+.newForm__errors {
```
Comment (YilKanda) [inline app/assets/stylesheets/components/_forms.scss]: The same here.

File: app/assets/stylesheets/components/_forms.scss [hunk 3]
```diff
@@ -67,27 +63,35 @@
...
+}
 
-  &__errors li::before {
-    color: red;
-    content: "\2022";
-    display: inline-block;
-    font-weight: bold;
-    width: 1rem;
-  }
+.newForm__errors li::before {
```
Comment (YilKanda) [inline app/assets/stylesheets/components/_forms.scss]: The same here

File: app/assets/stylesheets/components/_forms.scss [hunk 4]
```diff
@@ -67,27 +63,35 @@
...
+  color: red;
+  content: "\2022";
+  display: inline-block;
+  font-weight: bold;
+  width: 1rem;
+}
```
Comment (YilKanda) [inline app/assets/stylesheets/components/_forms.scss]: This class can be embedded inside the one you named newForm__errors

File: app/assets/stylesheets/components/_forms.scss (hunk 1 above)
Comment (DerekAyala) [reply app/assets/stylesheets/components/_forms.scss]: those classes used to be inside the .newform but to reuse them in the new forms I decide to move them outside of the .newform

File: app/assets/stylesheets/components/_forms.scss [hunk 5]
```diff
@@ -81,13 +81,20 @@
...
+      display: inline-block;
+      font-weight: bold;
+      width: 1rem;
+    }
   }
 }
+
+
+
+
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_forms.scss]: Please delete this extra spaces

File: app/views/devise/confirmations/new.html.erb [hunk 6]
```diff
@@ -1,16 +1,18 @@
...
+  <%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+    <div class="newForm__bg newForm__gap">
+      <div class="newForm__row--short">
+        <%= f.label :email, class: "newForm__label" %>
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email), class: "newForm__input" %>
+      </div>
 
-<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+      <div class="newForm__row">
+        <div></div>
```
Comment (JuanequeX) [inline app/views/devise/confirmations/new.html.erb]: What is the purpouse of this line?

File: app/views/devise/confirmations/new.html.erb [hunk 7]
```diff
@@ -1,16 +1,18 @@
-<h2>Resend confirmation instructions</h2>
+<div class="margin-top"></div>
```
Comment (JuanequeX) [inline app/views/devise/confirmations/new.html.erb]: Adding a div only to add a top margin is not a good practice, add it in the correct component, in all yours html.erb files

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/views/devise/confirmations/new.html.erb (hunk 6 above)
Comment (DerekAyala) [reply app/views/devise/confirmations/new.html.erb]: Screenshot from 2022-02-23 20-48-34
to leave the button on one side.

Comment (JuanequeX) [reply app/views/devise/confirmations/new.html.erb]: mmmm... i think that this a bad way to solve it, why not try using flexbox?

**Review by kevinnio — APPROVED** [review]

**Review by JuanequeX — APPROVED** [review]
LGTM

**Pull request merged**